klog.go

```diff
@@ -585,9 +591,10 @@ func (l *loggingT) header(s severity, depth int) (*buffer, string, int) {
 		file = "???"
 		line = 1
 	} else {
-		slash := strings.LastIndex(file, "/")
-		if slash >= 0 {
-			file = file[slash+1:]
+		dir := path.Base(path.Dir(file))
+		file = path.Base(file)
+		if l.addDirHeader {
+			file = path.Join(dir, file)
```

Comment:
alternatively, use a second `strings.LastIndex` call to back up a second segment if the addDirHeader option was true, and skip calling path.Base/path.Dir entirely

Reply:
@liggitt what happens on windows? :)

Reply:
what happens today? `/` is hard-coded

Reply:
also, this is using `path.Base(path.Dir(...))`, not filepath...

Reply:
ah good point.

Reply:
Using a second LastIndex also sounds good. For windows compatibility question, we should probably open another issue